portfolio/src/main/webapp/style.css

```diff
@@ -378,6 +378,15 @@
     font-weight: 600;
     color: yellow;
   }
+
+  form{
```

Comment:
Leave a single between a declarations and the opening brace: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation